swift/generate-short-url/README.md

```diff
@@ -0,0 +1,79 @@
+# 🔗 Get Short Url
+
+A sample Swift Cloud Function for generating a Short URL using [tinyurl](https://tinyurl.com/app) and [bitly](https://bitly.com/) APIs
+
+
+_Example input:_
+
+```json
+// For bitly
+
+{
+    "provider": "bitly",
+    "url": "https://www.google.com"
+}
+```
+
+```json
+// For tinyurl
+
+{
+    "provider": "tinyurl",
+    "url": "https://www.google.com"
+}
+```
+
+
+_Example output:_
+
+
+```json
+// Success
+
+{
+    "success": true,
+    "url": "https://bit.ly/3yvTt7c",
+}
+```
+
+```json
+// Error
+
+{
+    "success": false,
+    "message": "Entered URL is not a valid URL."
+}
+```
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud function:
```

Comment:
Let's remove references to environment since variables are no longer environment variables:

```suggestion
## 📝 Variables

List of variables used by this cloud function:
```